site2/docs/deploy-bare-metal-multi-cluster.md

```diff
@@ -321,14 +321,6 @@ You can also start brokers in the foreground by using [`pulsar broker`](referenc
 $ bin/pulsar broker
 ```
 
-## Service discovery
```

Comment:
I think this section needs to be revised, not removed. For example, `Pulsar provides a built-in service discovery mechanism that you can set up using the instructions immediately below.` could be removed, since there is not a built in option for this discovery. However, since users still have to solve the discovery problem, I think this section is still relevant. Now that the `Discovery Service` is removed, users have two options: implement their own discovery, or rely on an orchestration system, like kubernetes, that implements discovery for them.

Reply:
👍